src/effects/backends/builtin/builtinbackend.cpp

```diff
@@ -88,9 +88,7 @@ const QList<QString> BuiltInBackend::getEffectIds() const {
 }
 
 EffectManifestPointer BuiltInBackend::getManifest(const QString& effectId) const {
-    VERIFY_OR_DEBUG_ASSERT(m_registeredEffects.contains(effectId)) {
-        return EffectManifestPointer();
-    }
+    // This may return a null pointer in case a previously stored effect is no longer available
```

Comment:
Please move this to a `///` documentation comment in the header so this information is easier to find for the caller.